frontend/src/hooks/api/getters/useProjectOverview/useProjectOverview.ts

```diff
@@ -50,4 +50,12 @@ export const useProjectOverviewNameOrId = (id: string): string => {
     return useProjectOverview(id).project.name || id;
 };
 
+export const featuresCount = (
```

Comment:
backend doesn't return all features for the overview, but it already counts features by feature type so we can sum all types